Jellyfin.Server/Migrations/Routines/MigrateKeyframeData.cs

```diff
@@ -63,48 +56,33 @@ public MigrateKeyframeData(
     /// <inheritdoc />
     public void Perform()
     {
-        const int Limit = 100;
-        int itemCount = 0, offset = 0, previousCount;
+        const int Limit = 5000;
+        int itemCount = 0, offset = 0;
 
         var sw = Stopwatch.StartNew();
-        var itemsQuery = new InternalItemsQuery
-        {
-            MediaTypes = [MediaType.Video],
-            SourceTypes = [SourceType.Library],
-            IsVirtualItem = false,
-            IsFolder = false
-        };
 
         using var context = _dbProvider.CreateDbContext();
+        var records = context.BaseItems.Where(b => b.MediaType == MediaType.Video.ToString() && !b.IsVirtualItem && !b.IsFolder).Count();
+        _logger.LogInformation("Checking {Count} items for importable keyframe data.", records);
+
         context.KeyframeData.ExecuteDelete();
         using var transaction = context.Database.BeginTransaction();
-        List<KeyframeData> keyframes = [];
-
         do
         {
-            var result = _libraryManager.GetItemsResult(itemsQuery);
-            _logger.LogInformation("Importing keyframes for {Count} items", result.TotalRecordCount);
-
-            var items = result.Items;
-            previousCount = items.Count;
-            offset += Limit;
-            foreach (var item in items)
+            var results = context.BaseItems.Where(b => b.MediaType == MediaType.Video.ToString() && !b.IsVirtualItem && !b.IsFolder).OrderBy(e => e.Id).Skip(offset).Take(Limit).Select(b => new Tuple<Guid, string?>(b.Id, b.Path)).ToList();
```

Comment:
Minor thing, but you should store the "base" IQueryable in a variable so any query you make uses the same filter. As it is now, the Count and the Select filters can drift.